tests/new_unit_tests/Commands/Connection/InfoCommandTest.php

```diff
@@ -0,0 +1,206 @@
+use Pantheon\Terminus\Config;
+
+use Prophecy\Prophet;
+use Terminus\Models\Environment;
+use Terminus\Models\Site;
+use VCR\VCR;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Connection\InfoCommand
+ */
+class InfoCommandTest extends ConnectionCommandTest
+{
+    private $prophet;
+
+    /**
+     * Test suite setup
+     *
+     * @return void
+     */
+    protected function setup()
+    {
+        parent::setUp();
+
+        // TODO: (ajbarry) Where is the appropriate place to configure VCR mode?
+        // Picking up 'new_episodes' somewhere
+        VCR::configure()->setMode('none');
+
+        $this->command = new InfoCommand(new Config());
+        $this->prophet = new Prophet;
+    }
+
+    protected function tearDown()
+    {
+        parent::tearDown();
+
+        $this->prophet->checkPredictions();
+    }
+
+
+    /**
+     * Exercises connection:info command with a valid environment
+     *
+     * @return void
+     *
+     * @vcr site_connection-info
+     */
+    public function testConnectionInfo()
+    {
+        // command output with a valid site
+        $out = $this->command->connectionInfo('behat-tests.dev');
+
+        // should return a RowOfFields object
+        $this->assertInstanceOf(RowsOfFields::class, $out);
+
+        // should have a field structure
+        $connection_info = $out->getArrayCopy();
+        $this->assertEquals(['env', 'param', 'value'], array_keys($connection_info[0]));
+
+        // should contain connection parameters
+        $parameters = array_column($connection_info, 'param');
+        $this->assertContains('sftp_command', $parameters);
+        $this->assertContains('git_command', $parameters);
+        $this->assertContains('mysql_command', $parameters);
+        $this->assertContains('redis_command', $parameters);
+    }
+
+    /**
+     * Exercises connection:info command with a valid environment and filter
+     *
+     * @return void
+     *
+     * @vcr site_connection-info
+     */
+    public function testConnectionInfoFilter()
+    {
+        // command output using a filter argument
+        $out        = $this->command->connectionInfo('behat-tests.dev', 'git_command');
+        $parameters = array_column($out->getArrayCopy(), 'param');
+
+        // assert parameter count
+        $this->assertCount(1, $parameters);
+
+        // assert filtered field
+        $this->assertContains('git_command', $parameters);
+    }
+
+    /**
+     * Exercises connection:info command without a valid environment argument
+     *
+     * @return void
+     */
+    public function testConnectionInfoInvalid()
+    {
+        $this->markTestIncomplete(
+            'Skipping testConnectionInfoInvalid() -- Needs work'
+        );
+
+        $this->setInput([
+            'command' => 'connection:info',
+            // TODO: This does not appear to be the correct way to specify the command arguments
+            ['invalid-env'],
+        ]);
```

Comment:
If there was an option, you would literally use `--option` or `--option=value` (or put the value in the next item of the array, as needed).